### PR TITLE
Smashing magazine Clone

### PR DESCRIPTION
**Pull request by Laguna1**
Working on this project we build a heat map of the Smashing Magazine page.
Please check our code and let us know if everything is fine.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,48 @@
...
+The HTML was validated with W3C validation checker and the CSS styling was verified locally using a CSS stylelinter. All errors were corrected. 
+
 # Smashing-Magazine-Clone
 Project: Design Teardown
+
+## BUILT WITH
+*  HTML
+*  CSS
+***
+ ![Example]()
```
Comment (euqueme) [inline README.md]: line #7 the image is broken please provide an image or delete this line

File: README.md [hunk 2]
```diff
@@ -1,2 +1,48 @@
...
+
+### Live Demo  
+ [Smashing-Magazine-Clone]() - From the Odin Project Curriculum.
```
Comment (euqueme) [inline README.md]: please add a live demo on line #20

File: index.html [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+    <!-- #endregion White Cards-->
+
+    <!-- #region People Images -->
+
+    <div class="white_block clearme">SEE ALL LATEST ARTICLES</div>
+    <div class="lightgreen_block">Ethical Design Handbook:<br>
+        Leaving Dark Patterns Behind.</div>
+    <div class="white_block container bg_cont">
+        <div class="image">image</div>
+        <div class="small_art">
```
Comment (euqueme) [inline index.html:93]: Change the `<div>` tag for an` <article>` tag in line #90

File: index.html [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+        <div class="container">
+            <div class="image_small">image</div>
+            <div class="small_art_2">
+                <h2>PERSON OF THE WEEK</h2>
+                <span>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Dolor, ab?</span>
+            </div>
+        </div>
+        <div class="container">
+            <div class="image_small">image</div>
+            <div class="small_art_2">
```
Comment (euqueme) [inline index.html:108]: Change the `<div>` tag for an` <article>` tag in line #105

File: index.html [hunk 5]
```diff
@@ -0,0 +1,237 @@
...
```
Comment (euqueme) [inline index.html:115]: Change the `<div>` tag for an` <article>` tag in line #112

File: index.html [hunk 6]
```diff
@@ -0,0 +1,237 @@
...
```
Comment (euqueme) [inline index.html:122]: Change the `<div>` tag for an` <article>` tag in line #119

File: index.html [hunk 7]
```diff
@@ -0,0 +1,237 @@
...
```
Comment (euqueme) [inline index.html:129]: Change the `<div>` tag for an` <article>` tag in line #126

File: index.html [hunk 8]
```diff
@@ -0,0 +1,237 @@
...
```
Comment (euqueme) [inline index.html:136]: Change the `<div>` tag for an` <article>` tag in line #133

File: index.html [hunk 9]
```diff
@@ -0,0 +1,237 @@
...
+
+        <div class="i_design_bookimage">Book Overflow and absoluteposition </div>
+
+    </div>
+
+    <!-- #endregion Design Systems-->
+
+    <!-- #region Curated Posts -->
+    <div class="curated_posts">
+        <div class="challenges">
```
Comment (euqueme) [inline index.html]: Change the `<div>` tag for an` <article>` tag in line #161

File: index.html [hunk 10]
```diff
@@ -0,0 +1,237 @@
...
+            <h1>Heading</h1>
+            <div>Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum minus cum libero recusandae debitis
+                nesciunt. Corporis alias ex veniam aut. Nemo magni repellat rem quos facere voluptas perspiciatis
+                similique cupiditate. Alias, ad ex eligendi aperiam repudiandae quas nam debitis voluptates!
+            </div>
+            <div class="optimisations">
```
Comment (euqueme) [inline index.html]: Change the `<div>` tag for an` <article>` tag in line #167

File: index.html [hunk 11]
```diff
@@ -0,0 +1,237 @@
...
+                <h1>Heading</h1>
+                <div>Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum minus cum libero recusandae debitis
+                    nesciunt. Corporis alias ex veniam aut. Nemo magni repellat rem quos facere voluptas perspiciatis
+                    similique cupiditate. Alias, ad ex eligendi aperiam repudiandae quas nam debitis voluptates!</div>
+            </div>
+            <div class="workflow">
```
Comment (euqueme) [inline index.html]: Change the `<div>` tag for an` <article>` tag in line #173

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the Review by Maria Eugenia Queme P. (TSE)

### Hi @Laguna1 and @Romel4321  :lollipop: :balloon: 

### This is a great effort :muscle: I think you have a good understanding of the Smashing Magazine focal points :medal_sports: 

### Still there are a few more aspects to achieve :handshake: 

## Page structure

**Navigation bar**

- [x] It uses `<div> `and positions elements as at the source page (content is not important). lets add the 7 navbar elements

image

**Article tag**

- [x] It uses [`article` tag](https://www.w3schools.com/tags/tag_article.asp) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/design_teardown/article_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59540025/)), I've commented your code where the `<article> `tag should replace `<div>` tag

**Footer**

- [x] It contains centered links (using `<a>` tags) (content is not important)

image

## Playing with Typography

- [x] In this project we also practice using Google fonts, please download at least 2 fonts that are similar to the smashing magazine fonts and use them in your project.

image

## Generals

**Live Demo**

- [x] Your live demo is currently taking me to a broken page, let's fix it!

image

[OPTIONAL] I encourage the use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

### That's it, team! :raised_hands: 

### Please :pray: do these changes :memo: and submit back for a code review when you are done :rocket: 

Happy Coding! :woman_technologist: 

Any question? reach to me on slack **@ Maru**

PROJECT STATUS: REQUESTED CHANGES :vertical_traffic_light:

**Comment by Laguna1** [comment]
Demo link will work after we merge branches

**Comment by addod19** [comment]
## Pull request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 12]
```diff
@@ -0,0 +1,251 @@
...
+    <!-- #region Main Navbar -->
+    <nav >
+        <ul class="main_navbar">
+            <li class="nav_elements">Block One</li>
+            <li class="nav_elements">Block Two</li>
+            <li class="nav_elements">Block THree</li>
+            <li class="nav_elements">Block fcour</li>
+            <li class="nav_elements">Block Five</li>
+            <li class="nav_elements">Block Six</li>
+            <li class="nav_elements">
```
Comment (addod19) [inline index.html]: nrev3

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes 🔴

Hi @Laguna1 and @Romel4321, you have put a great effort into this project. 👍 However, we still need to fix the comments requested by the previous reviewer. 
- Kindly fix this comment
nrev1

Kindly make these changes and submit another code review 😄 
Good luck with your next submission 👍 
Happy coding 💻 
Daniel Larbi Addo

**Comment by Laguna1** [comment]
We changed `nav`, now it contains `div`
https://github.com/Laguna1/Smashing-Magazine-Clone/blob/761ba0de2183110e7d9e0355e888f1c3bc7091e2/index.html#L16

File: index.html [hunk 13]
```diff
@@ -0,0 +1,251 @@
...
+    <!-- #region Four Square Blocks -->
+    <div class="block_one">
+        <div class="block_zero">DON`T MISS THESE LATEST<br> ARTICLES</div>
+        <div class="block_one first">LOREM IPSUM</div>
+        <div class="block_one first">LOREM IPSUM</div>
+    </div>
+    <div class="block_two block_one">
+        <div class="block_two second">LOREM IPSUM</div>
+        <div class="block_two second">LOREM IPSUM</div>
+    </div>
```
Comment (addod19) [inline index.html:41]: ### [OPTIONAL] 2 ARTICLE ROWS
arti

File: index.html [hunk 14]
```diff
@@ -0,0 +1,251 @@
...
+        <div class="red_card-start">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusamus numquam
+            nam inventore itaque nostrum quae incidunt alias nisi? Corporis iusto mollitia, hic amet aliquam minima
+            recusandae voluptatibus sequi nisi eaque!</div>
+        <div class="mywhite_card">
+            Lorem ipsum dolor sit amet, consectetur adipisicing elit. Accusamus numquam
+            nam inventore itaque nostrum quae incidunt alias nisi? Corporis iusto mollitia, hic amet aliquam minima
+            recusandae voluptatibus sequi nisi eaque!
+        </div>
+    </div>
+
```
Comment (addod19) [inline index.html:66]: ### [OPTIONAL] #LINE 42-76
arti2

File: index.html [hunk 15]
```diff
@@ -0,0 +1,251 @@
...
+        <div class="white_card-one one">LOREM IPSUM</div>
+        <div class="white_card-two one ">LOREM IPSUM</div>
+        <div class="white_card-three one">LOREM IPSUM</div>
+    </div>
+
+
+    <div class="big_red_divider">
+        <div>Large Carrtoon Image</div>
+
+    </div>
```
Comment (addod19) [inline index.html:82]: arti3

**Review by addod19 — APPROVED** [review]
## Status: Approved 🎈 

Congratulations fixing the changes recommended by the previous reviewer 👍 . However, there are some optional comments I would like you to change before merging to master. Kindly fix it.
I would have to approve your project.
Good luck with your next project. 🙂 
Happy coding 💻 
Daniel Larbi Addo

**Pull request merged**